app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -126,6 +127,7 @@ public function update(Request $request, Campaign $campaign)
         $this->validate($request, [
             'internal_title' => 'string',
```

Comment:
This is missing the `unique:campaigns` validation rule that we use when creating a new campaign.

Reply:
You can include it here [by specifying a campaign ID to ignore for the uniqueness check](https://laravel.com/docs/5.5/validation#rule-unique), e.g:

```php
'internal_title' => [Rule::unique('campaigns')->ignore($campaign->id)],
```

Reply:
very cool, thanks @DFurnes !